### PR TITLE
Convert Carbon-Bot to Rasa 2.x

### LINKED ISSUE #38: Convert Carbon-Bot to Rasa 2.x
### Goal
Get CB working in `rasa shell` under Rasa 2.x. The new version should be as faithful to original as possible because we want to benchmark any subsequent changes. This is not completely possible however as some policies that were in use are now deprecated.

Updates happening on `phoenix-bot` branch.

### Tasks to complete goal
- [x] Run `rasa data convert nlu|nlg|core -f yaml --data={SOURCE_DIR} --out={TARGET_DIR}`
- [x] Run `rasa train` and record warnings and errors. 
- [x] Fix missing conveRT url exception
- [x] Convert unfeaturized slot types to ~~`influence_conversation: false`~~ `any`
- [x] Move responses to domain file.
- [x] Update forms to 2.0
- [x] Add action_calculate_offsets actions to train/test stories
- [ ] ~~(Optionally) FIx stopover slot filling in `airtravel_form`.~~ decided against this for now.
- [x] Fix entities that are not slots but treated like slots.
- [x] Convert test stories to 2.0 format/sort out why they make the evaluation crash.
- [x] Clean up and organize training stories. ~Figure out why there are duplicate slot setting events.~
- [ ] ~~Use test story to performance to determine if the bot is working sufficiently similar to its original behavior.~~ bot performs well on the test set, and seems to behave similarly to original behaviour... but the test set is small. More rigorous investigation of this would probably mean creating a new test set by digging through old rasa x conversations
- [x] Update Rasa and Rasa X versions in the requirements data.

### PR DESCRIPTION
**Pull request by koernerfelicia**
Closes #38

### CODE REVIEW

File: actions.py [hunk 1]
```diff
@@ -161,7 +162,6 @@ def calculate_emissions(self, tracker, unit="short_tons"):
         departure_iata = tracker.get_slot("iata_departure")
         destination_iata = tracker.get_slot("iata_destination")
         flight_class = tracker.get_slot("travel_flight_class")
-        stopover_iata = tracker.get_slot("iata_stopover")
```
Comment (tttthomasssss) [inline actions.py:164]: Have we removed the whole stopover thing altogether?

**Review by tttthomasssss — APPROVED** [review]
Awesome work 🎸  - good to go imho!

File: actions.py (hunk 1 above)
Comment (koernerfelicia) [reply actions.py:164]: I removed it for this update, yes. Chris had commented it out, but I don't like to keep commented-out code (that's why we have git 😅).

There were a couple of other features where it wasn't clear if they were finished and how well they worked, like reference resolution of cities, correcting cities, and stopover. Maybe I should make issues to remind us that these existed? What do you think?

Comment (tttthomasssss) [reply actions.py:164]: Yes, I'd say issues for any of these will be a good idea so we have a record of them on the books.

**Pull request merged**